src/pages/CreatedReviewManagePage/index.tsx

```diff
@@ -1,25 +1,71 @@
 //생성한 리뷰 관리 페이지
-import { useSendReview } from '@/apis/hooks'
+import { Suspense } from 'react'
+import { useLocation } from 'react-router-dom'
+import { Header } from '@/components'
+import { useGetReviewQuestion } from '@/apis/hooks'
+import useActiveTab from '@/pages/CreatedReviewManagePage/hooks/useReviewTab'
```

Comment:
```suggestion
import useActiveTab from './hooks'
```

이렇게 불러오게끔 hooks 폴더에 index 파일을 만들면 좋을 것 같습니다